src/js/components/AppVersionListComponent.jsx

```diff
@@ -163,17 +163,23 @@ var AppVersionListComponent = React.createClass({
   },
 
   render: function () {
+    var currentVersion = AppsStore.getCurrentApp(this.props.appId);
+
+    var versionDate = currentVersion.version != null ?
+      ` - ${new Date(currentVersion.version).toLocaleString()}` :
```

Comment:
Minor quibble, but can we move this out into its own variable? IMO it's a bit ugly in a template string...